src/server/lsHost.ts

```diff
@@ -171,12 +171,16 @@ namespace ts.server {
             return this.host.fileExists(path);
         }
 
+        readFile(fileName: string): string {
+            return this.host.readFile(fileName);
+        }
+
         directoryExists(path: string): boolean {
             return this.host.directoryExists(path);
         }
 
-        readFile(fileName: string): string {
-            return this.host.readFile(fileName);
+        readDirectory(path: string, extensions?: string[], exclude?: string[], include?: string[]): string[] {
+            return this.host.readDirectory(path, exclude, exclude, include);

```

Comment:
This doesn't look right. Did you mean to use `exclude` twice, and not use `extensions`? If so, add a quick comment to note that it's deliberate.